webapp-django/crashstats/crashstats/views.py

```diff
@@ -782,11 +778,12 @@ def crashes_per_day(request, default_context=None):
     # to be called `version`
     supersearch_params = copy.deepcopy(params)
     supersearch_params['version'] = supersearch_params.pop('versions')
-    supersearch_params['platform'] = supersearch_params.pop('platforms')
-    # in SuperSearch it's called 'Mac' not 'Mac OS X'
-    if 'Mac OS X' in supersearch_params['platform']:
-        supersearch_params['platform'].append('Mac')
-        supersearch_params['platform'].remove('Mac OS X')
+    if 'platforms' in supersearch_params:
+        supersearch_params['platform'] = supersearch_params.pop('platforms')
+        # in SuperSearch it's called 'Mac' not 'Mac OS X'
+        if 'Mac OS X' in supersearch_params['platform']:
+            supersearch_params['platform'].append('Mac')
+            supersearch_params['platform'].remove('Mac OS X')
```

Comment:
This changes the behavior such that if the user didn't specify platforms, then it doesn't reduce on platforms at all.

This seems like a big backwards-incompatible behavioral change. Is this really what we want to do? Any idea how it'll affect the crashes-per-day graphs? Should we tell someone that this is happening before we land it?

Would it be better to add a "Ignore platform" option instead?